test/e2e/landing-page.spec.ts

```diff
@@ -1,10 +1,12 @@
 import { test, expect } from '@playwright/test';
+import { prisma } from '../../app/db.server';
 import { loggedInAsUser } from './utils';
 
 test.describe('Landing Page', () => {
   loggedInAsUser();
 
   test.beforeEach(async ({ page }) => {
+    await prisma.certificate.deleteMany({});
```

Comment:
Why does `deleteMany` require you to pass an empty object literal?  Can't we do that in the definition of the function?